Makefile

```diff
@@ -177,10 +177,11 @@ export KUBEBUILDER_ASSETS=$(PROJECT_DIR)/bin
 
 .PHONY: test
 test: generate-mocks lint bin/ginkgo bin/kubectl bin/kube-apiserver bin/etcd ## Run tests. At the moment this is only unit tests.
+	@./hack/testing_ginkgo_recover_statements.sh --add # Add ginkgo.GinkgoRecover() statements to controllers.
```

Comment:
This does have to happen prior to the test suite process as the controllers are imported before the test suite can do anything about it.